tests/selenium/cookie_test.py

```diff
@@ -17,6 +23,48 @@
 class CookieTest(pbtest.PBSeleniumTest):
     """Basic test to make sure the PB doesn't mess up with the cookies."""
 
+    @unittest.skipIf(os.environ.get('BROWSER') == 'firefox', """
+Disabled until Firefox fixes bug:
+https://github.com/EFForg/privacybadger/pull/1347#issuecomment-297573773""")
+    def test_dnt_check_should_not_set_cookies(self):
+        TEST_DOMAIN = "dnt-test.trackersimulator.org"
+        TEST_URL = "https://{}/".format(TEST_DOMAIN)
+
+        RUN_DNT_JS = """badger.checkForDNTPolicy(
+  '{}', 0, r => window.DNT_CHECK_RESULT = r
+);""".format(TEST_DOMAIN)
+        CHECK_FINISHED_JS = "return window.DNT_CHECK_RESULT === false"
+
+        # verify that the domain itself doesn't set cookies
+        self.load_url(TEST_URL)
+        self.assertEqual(len(self.driver.get_cookies()), 0,
+            "No cookies initially")
+
+        # directly visit a DNT policy URL known to set cookies
+        self.load_url(TEST_URL + ".well-known/dnt-policy.txt")
+
```

Comment:
Should we verify the cookie here instead of after loading `TEST_URL` again below?